### PR TITLE
fix: prevent content from breaking mobile layout

### PR DESCRIPTION
**Pull request by ryan-ludwig**
Following up on an email from Google's search console today, 

```
Content wider than screen
We recommend that you fix these issues when possible to enable the best experience and coverage in Google Search.
```

Screen Shot 2021-11-22 at 10 16 52 AM

This doesn't fix the root problem, something's odd with the content coming in and adding a bunch of `&nbsp;`
image

but it does prevent the layout from breaking in those cases.

https://www.kiva.org/lend-beta/2272570

### CODE REVIEW

File: src/components/BorrowerProfile/DetailsTabs.vue [hunk 1]
```diff
@@ -287,8 +287,8 @@ export default {
 				this.loan.loanTermLenderRepaymentTerm = loan?.terms?.lenderRepaymentTerm ?? 0;
 				this.loan.lossLiabilityCurrencyExchange = loan?.terms?.lossLiabilityCurrencyExchange ?? '';
 				this.loan.repaymentInterval = loan?.repaymentInterval ?? '';
-				this.loan.disbursalDate = loan.disbursalDate ?? '';
-				this.loan.status = loan.status ?? '';
+				this.loan.disbursalDate = loan?.disbursalDate ?? '';
+				this.loan.status = loan?.status ?? '';
```
Comment (ryan-ludwig) [inline src/components/BorrowerProfile/DetailsTabs.vue:291]: These are unrelated, just fixing console errors when pulling up the page on my vm.

File: src/components/BorrowerProfile/MoreAboutLoan.vue [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <template>
 	<section>
-		<div class="tw-prose">
+		<div class="tw-prose tw-overflow-auto">
```
Comment (mcstover) [inline src/components/BorrowerProfile/MoreAboutLoan.vue:3]: Seems ok. Also wonder if there is some "text-wrap" property that should be set here..

**Review by mcstover — APPROVED** [review]

File: src/components/BorrowerProfile/MoreAboutLoan.vue (hunk 2 above)
Comment (mcstover) [reply src/components/BorrowerProfile/MoreAboutLoan.vue:3]: `overflow-wrap: break-word;` could also work

Comment (ryan-ludwig) [reply src/components/BorrowerProfile/MoreAboutLoan.vue:3]: Nice!

**Review by mcstover — APPROVED** [review]
